src/common.h

```diff
@@ -125,7 +125,10 @@ int ishexn(const char *q, int n);
 int ishexucn(const char *q, int n);
 int ishexlcn(const char *q, int n);
 /* length of hex. if extra_chars not null, it will be 1 if there are more
- * non-hex characters after the length of valid hex chars returned. */
+ * non-hex characters after the length of valid hex chars returned.
+ * NOTE, the return will always be an even number (rounded down). So if we
+ * want the length of "ABCDE", it will be 4, not 5.
+ */
 size_t hexlen(const char *q, int *extra_chars);
```

Comment:
This too is fine but unrelated to the PR

Reply:
Agreed,  This is an easy pull out, and can be placed into a cleanup PR.